R/typeformAddin.R

```diff
@@ -0,0 +1,131 @@
+#' Questionnaire selection
+#'
+#' This rstudio add in is for selecting the appropriate questionnaire from the group of
+#' questionnaires available for a given api key based on matching the questionnaire name
+#' up with the unique id.
+#'
+#' @inheritParams get_typeforms
+#' @import miniUI
+#' @import shiny
+#' @importFrom rstudioapi insertText
+#' @export
+typeformAddin = function(api = NULL){
+  # a modified version of get_api since if there is no key stored or provided
+  # we want the user to be able to enter one.
+  api_flag = TRUE
+  api = if(is.null(api))
```

Comment:
Use get_api() here

Reply:
i avoided get_api() intentionally since that would then mean having to trap the error